db/migrate/20180607144007_create_remote_settings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRemoteSettings < ActiveRecord::Migration[5.1]
+  def change
+    create_table :remote_settings do |t|
+      t.string   "name", null: false
+      t.string   "url", null: false
+      t.text     "contents", null: false
+      t.timestamps
+    end
+    add_index :remote_settings, ["name"], name: "index_remote_settings_on_name", unique: true, using: :btree
+  end
```

Comment:
Is this longform index addition necessary? Doesn't Rails automatically give it the proper name when you do this:
```ruby
add_index :remote_settings, :name, unique: true
```
The default Postgres index type is btree, so I don't think it's necessary to specify. If you look at our `schema.rb`, it doesn't mention `btree` anywhere. See https://github.com/rails/rails/pull/27981

Reply:
Also, did you run this migration locally to make sure the `safe_migrations` gem didn't find any issues with it?

Reply:
Yes I ran the migration locally and it ran with no errors.  Did you encounter errors?

Reply:
Yes I normally don't add btree because it is the default.  However a majority of the prior migrations did.  So I thought they were just trying to make it clear.